project/assets/scripts/controller/details/details.js

```diff
@@ -1,49 +1,52 @@
-const DetailsEvents = function (options = {}) {
-    this.init = function (redirectToFunction) {
-        setNavEvent('new-evaluation', addEvent.bind(null, 'newEvaluation', redirectToFunction));
-        setNavEvent('evaluations', addEvent.bind(null, 'evaluations', redirectToFunction));
-        setNavEvent('logout', addEvent.bind(null, 'login', redirectToFunction));
-    };
-    this.destroy = function (redirectToFunction) {
-        removeEvents('new-evaluation', addEvent.bind(null, 'newEvaluation', redirectToFunction));
-        removeEvents('evaluations', addEvent.bind(null, 'newEvaluation', redirectToFunction));
-        removeEvents('logout', addEvent.bind(null, 'login', redirectToFunction));
-    };
-    this.populate = function (currentName) {
-        const currentCandidateData = getCurrentCandidateData(currentName);
-        console.log(currentName);
-        document.getElementById('candidate').value = currentCandidateData[0].inputData.candidate;
-        document.getElementById('interviewer').value = currentCandidateData[0].inputData.interviewer;
-        document.getElementById('date').value = currentCandidateData[0].inputData.date;
+(function () {
+    const DetailsEvents = function (options = {}) {
+        this.init = function (redirectToFunction) {
+            setNavEvent('new-evaluation', addEvent.bind(null, 'newEvaluation', redirectToFunction));
+            setNavEvent('evaluations', addEvent.bind(null, 'evaluations', redirectToFunction));
+            setNavEvent('logout', addEvent.bind(null, 'login', redirectToFunction));
+        };
+        this.destroy = function (redirectToFunction) {
+            removeEvents('new-evaluation', addEvent.bind(null, 'newEvaluation', redirectToFunction));
+            removeEvents('evaluations', addEvent.bind(null, 'newEvaluation', redirectToFunction));
+            removeEvents('logout', addEvent.bind(null, 'login', redirectToFunction));
+        };
+        this.populate = function (currentId) {
+            const currentCandidateData = getCurrentCandidateData(currentId);
+            document.getElementById('candidate').value = currentCandidateData.inputData.candidate;
+            document.getElementById('interviewer').value = currentCandidateData.inputData.interviewer;
+            document.getElementById('date').value = currentCandidateData.inputData.date;
 
-        document.getElementById(currentCandidateData[0].technicalLevel).checked = true;
-        document.getElementsByName('level').forEach((el) => {
-            const elem = el;
-            if (!elem.checked) {
-                elem.disabled = true;
-            }
-        });
+            document.getElementById(currentCandidateData.technicalLevel).checked = true;
+            document.getElementsByName('level').forEach((el) => {
+                const elem = el;
```

Comment:
Any particular reason why you're aliasing `el` as `elem`?

Reply:
ESLint underlined it if not, saying that I assign a value to parameter or something like this. That's why

Reply:
Crazy ESLint! :stuck_out_tongue: